pyaedt/edb_core/EDB_Data.py

```diff
@@ -587,6 +589,50 @@ def filling_material_name(self, value):
             self._filling_material_name = value
             self.update_layers()
 
+    @property
+    def negative_layer_value(self):
+        """Negative layer.
+
+        Returns
+        -------
+        bool
+            True if the layer is negative, else False.
+        """
+        if self._layer_type == 0 or self._layer_type == 2:
+            try:
+                self._negative_layer = self._layer.GetNegative()
+            except:
+                pass
+        return self._negative_layer
+
+    @negative_layer_value.setter
+    def negative_layer_value(self, value):
+        if self._layer_type == 0 or self._layer_type == 2:
+            self._negative_layer = value
+            self.update_layers()
+
+    @property
+    def roughness_enabled_value(self):
```

Comment:
```suggestion
    def roughness_enabled(self):
```
I would rename it as ``roughness_enabled``